src/Razor/test/RazorLanguage.Test/TestFiles/IntegrationTests/ComponentDesignTimeCodeGenerationTest/SingleLineControlFlowStatements_InCodeBlock/TestComponent.codegen.cs

```diff
@@ -45,7 +45,7 @@ protected override void BuildRenderTree(Microsoft.AspNetCore.Components.Renderin
 #line default
 #line hidden
 #nullable disable
-                                      
```

Comment:
I'm not sure why this happened. Haven't investigated it further yet. @NTaylorMullen any ideas off the top of your head?

Reply:
oh geez, no clue